README.md

```diff
@@ -1,33 +1,103 @@
 # overseas-entities-web
-Web front-end for the Register An Overseas Entity service
 
-### Requirements
+Web front-end for the **register an overseas entity and tell us about its beneficial owners** service. Link to the live page [here](https://www.gov.uk/guidance/register-an-overseas-entity)
+</br>
+
+## Overseas entities architecture
+
+![Overseas entities architecture](./docs/Overseas%20entities%20architecture.png)
+</br>
 
-In order to run the service locally you will need the following:
+## Frontend Technologies and Utils
 
-- [NodeJS](https://nodejs.org/en/)
+- [NodeJS](https://nodejs.org/)
+- [ExpressJS](https://expressjs.com/)
+- [Typescript](https://www.typescriptlang.org/)
+- [NunJucks](https://mozilla.github.io/nunjucks)
+- [GOV.UK Design System](https://design-system.service.gov.uk/)
+- [Jest](https://jestjs.io)
+- [Sonarqube](https://www.sonarqube.org)
+- [Docker](https://www.docker.com/)
+- [Tilt](https://tilt.dev/)
 - [Git](https://git-scm.com/downloads)
 
-### Getting started
+## Project Structure
+
+[Project Structure and Code Style](./docs/Project%20Structure%20and%20Code%20Style.md)
+
+## Running locally on Docker env
 
-To checkout and build the service:
+The only local development mode available, that includes account, redis and other important service/dependency is only possible through our development orchestrator service in [Docker CHS Development](https://github.com/companieshouse/docker-chs-development), that uses [tilt](https://tilt.dev/).
 
-1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README. 
-2. Run ./bin/chs-dev modules enable overseas-entities
-3. Run ./bin/chs-dev development enable overseas-entities-web (this will allow you to make changes).
-4. Run docker using "tilt up" in the docker-chs-development directory.
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README.
+2. Run `./bin/chs-dev modules enable overseas-entities`
+3. Run `./bin/chs-dev development enable overseas-entities-web` (this will allow you to make changes in real time).
+4. Run docker using `tilt up` in the docker-chs-development directory.
 5. Use spacebar in the command line to open tilt window - wait for overseas-entities-web to become green.
-6. Open your browser and go to page http://chs.local/register-an-overseas-entity
+6. Open your browser and go to page <http://chs.local/register-an-overseas-entity>
+
+Environment variables used to configure this service in docker are located in the file `services/modules/overseas-entities/overseas-entities-web.docker-compose.yaml`
+
+### Requirements
+
+1. node v12.22.12 (Soon it will be v16)
```

Comment:
`v16` has already been configured for concourse, an it is running in other environments for different services. It should be safely moved to the version 16 in the coming sprints.

Reply:
Agree but this is documenting what we currently have. Best not to guess what we might end up with in the coming sprints because we are.... AGILE! (and everything can change 😄 ),

Reply:
Please could this info from original comment be added:

_"... Concourse pipeline actually builds using Node 10 (I think) and live runs Node 12 too."_

Reply:
I thought I needed to remove just the future version. Anyway for development I do not think we need to add live and concourse version of node. But if you think will be of use I will add it :)

Reply:
👍 Thanks.